pkg/pool/operations/import.go

```diff
@@ -55,8 +55,10 @@ func (oc *OperationsConfig) Import(cspi *cstor.CStorPoolInstance) (bool, error)
 	devID := pool.GetDevPathIfNotSlashDev(bdPath[0])
 	cacheFile := types.CStorPoolBasePath + types.CacheFileName
 	if len(devID) != 0 {
+		// Cachefile property takes more precedence over the directory
+		// for sparse based pools importing based on directory make more
+		// sense. As if we mention WithCacheFile then directory is not considered
 		cmdOut, err = zfs.NewPoolImport().
-			WithCachefile(cacheFile).
```

Comment:
@kmova @sonasingh46 Removed `-c cachefile` option in the import command for sparse based pool. While building the command directory option will be ignored if cachefile is set.

Reply:
As discussed -- let us make the workflow :
1. Cache file import
2. Then, directory-based import if above fails

For all the pool type -- whether be sparse or disk.

Reply:
pushed the changes!! Verifying the same

Reply:
Changes look good.